Source/FieldSolver/WarpX_QED_Field_Pushers.cpp

```diff
@@ -152,15 +152,15 @@ WarpX::Hybrid_QED_Push (int lev, PatchType patch_type, Real a_dt)
         );
 
         // Make local copy of xi, to use on device.
-        const Real xi = WarpX::quantum_xi;
+        const Real xi_c2 = WarpX::quantum_xi_c2;
```

Comment:
this should be a `constexpr`

Reply:
The user can change this parameter at runtime so I don't think it can, right?

Reply:
oh right, mismatched. Sorry.